draft-pda-protocol.md

```diff
@@ -717,9 +723,9 @@ sub-requests by ignoring any sub-request that does not follow the previous one
 (See {{anti-replay}}.)
 
 The response is an HTTP 200 OK with a body consisting of the helper's updated
-state and a sequence of *sub-responses*, where the i-th sub-response corresponds
-to the i-th sub-request for each i. The structure of each sub-response is
-specific to the PPM protocol:
+state and a sequence of *sub-responses*, where the j-th sub-response corresponds
+to the j-th sub-request for each `1 <= j <= len(AggregateReq.seq)`. The
+structure of each sub-response is specific to the PPM protocol:
```

Comment:
This bit of mathiness seems a bit confusing. Instead of expanding the formula, I would consider "where each subresponse corresponds to the sub-request in the same position in AggregateReq"